README.md

```diff
@@ -230,6 +230,13 @@ A brief view of [__Principles in Construction__](Principles.md)
 
 Adaptive Education
 
+Education is one of the largest societal challenges for both basic functioning in a society and to gain important new skills to be able to contribute back. It needs to be addressed for both youth and continuing education and learning for all in the rapidly advancing world.
+- Democratizing education:
+	Education material can be seen as specialized format of knowledge for easy understanding and learning.
+	Wikipedia has been quite successful in creating a common store of knowledge which is community created, updated, and monitored. Many open source projects are sucessful using similar organization.
+	Similar techniques could be used to create a common source of education material which can be iteratively expanded and by increasing the ease of sharing, editing, and refining educational material.
```

Comment:
Yes!

The education material matter so much! I think the mother tree in Avatar is some thought of idealistic approach: individuals will download (learn), live their life (do/execute) and upload (share & aggregate)

It doesn't touch on how knowledge aggregation works, and simplified the "plug and play" knowledge sync process, but essentially, it's a good direction to explore!